app/views/users/edit.html.slim

```diff
@@ -0,0 +1,21 @@
+h1.d-flex.justify-content-center
+  |  Edit Profile
+hr
+.d-flex.justify-content-center
+  = form_with model:@user , url:"edit" , method: :put do |f|
```

Comment:
Man ir aizdomas kā šeit tu vispar vareji uzrakstit: = form_with @user do |f|
https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-form_with

Šis form_with ir ļoti viltigs, tāpāt ka Rails.